python/Ganga/Core/GangaRepository/SubJobXMLList.py

```diff
@@ -116,7 +116,7 @@ def load_subJobIndex(self):
 
         raw_self = stripProxy(self)
```

Comment:
Unfortunately this is done in many, many places (for better or worse).

I'm hoping to address it by removing/changing this everywhere with another branch dedicated to changing this as when I replaced ```stripProxy(self)``` with ```self``` with a macro I saw Ganga have a heart attack due to a few different things getting broken.